build.xml

```diff
@@ -610,6 +719,11 @@
         <available file="${contentdir}/test/sub" type="dir" />
       </not>
       <then>
+        <!-- avoid PHP Errors by creating each level separately -->
+        <mkdir dir="${themedir}" />
+        <mkdir dir="${themedir}/templates" />
+        <mkdir dir="${contentdir}" />
+        <mkdir dir="${contentdir}/test" />
```

Comment:
Would be nice if they had an equivalent to the `-p` switch!

Reply:
Agreed. It works like `-p`, but spits out scary-looking error messages (it did show warnings before as well, but not so scary-looking).